### PR TITLE
Add ecs input variable

### PR DESCRIPTION
**Pull request by yriahi**
Adds a new input variable for the ECS cluster module.

### CODE REVIEW

File: ecscluster/variables.tf [hunk 1]
```diff
@@ -39,6 +39,12 @@ variable "security_groups" {
   default     = []
 }
 
+variable "custom_cluster_policies" {
```
Comment (rbayliss) [inline ecscluster/variables.tf]: Nitpick - we just call this `policies` in the ASG module.  Can we call it `policies` here so it's consistent?

**Review by rbayliss — CHANGES_REQUESTED** [review]
Hey Youssef,
Good start, but don't we need to actually do something with the policies that are passed in (eg: pass them to the `asg` module)?

File: ecscluster/variables.tf (hunk 1 above)
Comment (yriahi) [reply ecscluster/variables.tf]: Happy to make that change.
I purposefully named it `custom_cluster_policies` to distinguish it from the ASG. Wouldn't one be overriding the other? or does the `policies` ARNs list apply to both?

Comment (yriahi) [reply ecscluster/variables.tf]: Input variable renamed.

**Pull request merged**